src/manifest/parsers/index.ts

```diff
@@ -114,11 +114,16 @@ export class ManifestParser {
   }
 
   private readAndParseManifest(projectRoot: Uri, targetPath: string) {
-    const manifestLocation = path.join(
+    let manifestLocation = path.join(
```

Comment:
Maybe change this into a check if the supplied path is relative vs absolute?

Reply:
Ya - good call. Can do.